pylib/anki/stats.py

```diff
@@ -103,13 +103,13 @@ class CollectionStats:
     def __init__(self, col) -> None:
         self.col = col
         self._stats = None
-        self.type = 0
+        self.type = CARD_TYPE_NEW
         self.width = 600
         self.height = 200
         self.wholeCollection = False
 
     # assumes jquery & plot are available in document
-    def report(self, type=0) -> str:
+    def report(self, type=CARD_TYPE_NEW) -> str:
```

Comment:
type is not a card type here

Reply:
Corrected.
I also corrected the comment of method CollectionStats.report since it seems that the type indicated in comment are inconsistent with the 3 buttons in the footer